imod/mf6/package.py

```diff
@@ -401,16 +403,18 @@ def __to_datetime(self, time, use_cftime):
 
     def clip_box(
         self,
-        time_min=None,
-        time_max=None,
-        layer_min=None,
-        layer_max=None,
-        x_min=None,
-        x_max=None,
-        y_min=None,
-        y_max=None,
-        state_for_boundary=None,
-    ) -> "Package":
+        time_min: Optional[str] = None,
```

Comment:
is str the type of t_min? Isn't there a more precise type that could be used? Maybe here some input checking can also be added to make sure t_min has a string value that makes sense.